### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'length') (at "Corporation")

### ISSUE THREAD

**Issue by emsk3r** [danielyxie/bitburner#3017]
### How did this happen?

Please fill this information with details if relevant.

- [X] Save file
- [ ] Minimal scripts to reproduce the issue
- [ ] Steps to reproduce

### Environment

* Error: TypeError: Cannot read properties of undefined (reading 'length')
* Page: Corporation
* Version: v1.4.0 (49de4a28)
* Environment: Production
* Platform: Steam
* UserAgent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) bitburner/1.0.0 Chrome/93.0.4577.82 Electron/14.0.2 Safari/537.36
* Features: lang=de cookiesEnabled=true doNotTrack=null indexedDb=true
* Source: n/a

### Save
[RECOVERY_BITBURNER_1645557042.json.gz](https://github.com/danielyxie/bitburner/files/8119640/RECOVERY_BITBURNER_1645557042.json.gz)

**Comment by ChrissiQ** [comment]
Hi @emsk3r,

Would you be able to describe what you were doing when your game crashed?  I tried loading up your recovery file but it was not obvious to me how to reproduce this crash.

Thanks!

**Comment by emsk3r** [comment]
Hi ChrissiQ,

err… in fact Bitburner was in the background while I did something else, then when I focused on Bitburner again I got the crash screen. I guess Bitburner maybe tried to do a autosave. It's been inside BN3.1 and this was the first crash at all. Awesome game, by the way. I tried the leftmost option of trying to continue the game and it worked out nicely.

Sorry, can't report more details :-/
